geometry_msgs/Vector3 pose
uint8 id